tests/taskcluster/bin/raptor_graph

```diff
@@ -261,7 +201,7 @@ getEmulator()
     Object.keys(config).forEach(function(raptorTask) {
       var definition = config[raptorTask].definition;
 
-      config[raptorTask].params.workerType = RAPTOR_WORKER_TYPE;
+      //config[raptorTask].params.workerType = RAPTOR_WORKER_TYPE;
```

Comment:
if this is commented out, we probably don't need it, not sure if it was kept here just in case.